demos/python_demos/whiteboard_inpainting_demo/README.md

```diff
@@ -0,0 +1,85 @@
+# Whiteboard Inpainting Demo
+
+This demo focuses on a whiteboard text overlapped by a person. The demo shows
+how to use the OpenVINO<sup>TM</sup> toolkit to detect and hide a person on a
+video so that all text on a whiteboard is visible.
```

Comment:
It seems to me that the demo is not really whiteboard-specific. Wouldn't it work with any background? If so, then it would be better to rename it to something more generic, e.g. `background_inpainting_demo` and rephrase the description to only use whiteboards as an example.

Reply:
Actually we did not find more appropriate name. `Background inpainting` is not suitable for the demo because our target is text on a white or black board behind a person.

Reply:
> our target is text on a white or black board behind a person.

Well... doesn't that make it the background? If it was something other than a blackboard, wouldn't the demo still work?

Reply:
It works independently of background's type. We just focused on a text. @snosov1, can we use the proposed name: `background_inpainting_demo`?

Reply:
I discussed it with @snosov1 and we decided to keep the name as-is.